docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -54,7 +54,7 @@ installation followed and the operating system.
     .. code-block:: bash
 
         cd <path-to-Fast-DDS-workspace>/src/fastddsgen
-        gradle assemble
+        ./gradlew assemble
```

Comment:
Since eProsima/Fast-DDS-Gen#129, new `gradle` versions are supported so the command `gradle assemble` is correct having installed `gradle` previously, as outlined in [Fast DDS-Gen installation guide](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#fast-dds-gen-installation). Nevertheless, this guide includes a note indicating the possibility of using the `gradlew` script provided within the repository in case of error or if the user does not want to install `gradle` dependency. Finally, this section starts assuming that the installation process has been followed, so the user should have read that note:

> If the steps outlined in the Installation Manual have been followed, then *Fast DDS*, *Fast CDR*, and
Fast-RTPS-Gen should be installed in the system.

For all the previous reasons, I do not consider that this suggestion should be merged. Maybe it could be helpful to provide the user with the link to the installation guide instead of providing the commands (that will fail if the dependency has not been installed).